onadata/apps/api/viewsets/open_data_viewset.py

```diff
@@ -171,7 +171,7 @@ def destroy(self, request, *args, **kwargs):
         return Response(status=status.HTTP_204_NO_CONTENT)
 
     @action(methods=['GET'], detail=True)
-    def schema(self, request, **kwargs):
+    def open_data_schema(self, request, **kwargs):
```

Comment:
The `drf-spectacular` package utilizes the `ViewSet.schema` attribute to retrieve schema information and this endpoint overrides that... We'll need to find another name for this endpoint to bypass the error with `drf-spectacular`